CRM/Utils/System/Standalone.php

```diff
@@ -643,6 +643,12 @@ public function sessionStart() {
     if (!$this->isUserExtensionAvailable()) {
       $session_cookie_name = 'SESSCIVISOFALLBACK';
     }
+    else {
+      $session_cookie_name = 'SESSCIVISO';
```

Comment:
by doing this does this mean L655 can be removed as well?

Reply:
Yes - thanks @seamuslee001 - I've removed it